app/uk/gov/hmrc/thirdpartyapplication/scheduled/ResponsibleIndividualVerificationRemovalJob.scala

```diff
@@ -74,7 +74,7 @@ class ResponsibleIndividualVerificationRemovalJob @Inject()(val lockKeeper: Resp
   }
 
   private def sendRemovalEmailAndRemoveRecord(verificationDueForRemoval: ResponsibleIndividualVerification) = {
-    val declineReason = "The Responsible Individual has not accepted the Terms of Use."
+    val declineReason = "The responsible individual did not accept the terms of use in 20 days."
```

Comment:
If we change the interval in the config file we will need to remember to change this message as well, might be a good idea to add a comment into the application.conf file to that effect?

Reply:
Ideally we would generate the message from the config value but given that the interval could be anything I'm not sure if there is an easy way to do it

Reply:
I've added a comment in the config file....